spec/negotiation.html

```diff
@@ -239,14 +239,13 @@ <h1>ResolveLocale ( _availableLocales_, _requestedLocales_, _options_, _relevant
               1. Let _optionsValue_ be the string _optionsValue_ after performing the algorithm steps to replace Unicode extension values with their canonical form per <a href="https://unicode.org/reports/tr35/#Canonical_Unicode_Locale_Identifiers">Unicode Technical Standard #35 LDML § 3.2.1 Canonical Unicode Locale Identifiers</a>, treating _key_ as `ukey` and _optionsValue_ as `uvalue` productions.
               1. If _optionsValue_ is the empty String, then
                 1. Let _optionsValue_ be *"true"*.
-            1. If _keyLocaleData_ contains _optionsValue_, then
-              1. If SameValue(_optionsValue_, _value_) is *false*, then
-                1. Let _value_ be _optionsValue_.
-                1. Let _supportedExtensionAddition_ be *""*.
+            1. If SameValue(_optionsValue_, _value_) is *false* and _keyLocaleData_ contains _optionsValue_, then
```

Comment:
The order of conditions is flipped. I assume it'll not be a problem, but are we sure it's not observable or otherwise an issue?

Reply:
Correct, neither SameValue nor List containment are observable.